templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -453,6 +456,14 @@ Parameters:
     ConstraintDescription: Must be 'General Purpose (SSD)' or 'Provisioned IOPS'.
     Description: Database storage type
     Type: String
+  ElasticSearchPassword:
+    Description: "Password for the elastic search master user, the username will be 'bitbucket'. Must be at least 8 characters and include 1 uppercase, 1 lowercase, 1 number, and 1 of the following symbols: ! # $ { * : [ = , ] - _ + % &"
+    Type: String
+    AllowedPattern: '(?=^.{6,255}$)((?=.*\\d)(?=.*[A-Z])(?=.*[a-z])|(?=.*\\d)(?=.*[^A-Za-z0-9])(?=.*[a-z])|(?=.*[^A-Za-z0-9])(?=.*[A-Z])(?=.*[a-z])|(?=.*\\d)(?=.*[A-Z])(?=.*[^A-Za-z0-9]))^.*'
```

Comment:
Is this enforced by elasticsearch or we just reuse the same pattern from db.

Reply:
Just re-using the pattern from the db.

Reply:
Btw, the regex will pass with 6 characters - the first qualifier tests the length and you can use `aB2$as` as a password. The same problem is with other passwords as well.
<img width="1128" alt="Screen Shot 2021-02-25 at 11 47 10 am" src="https://user-images.githubusercontent.com/416238/109086616-063a9380-7760-11eb-9366-434d3a50b7d5.png">

Reply:
Yep but the cloud formation `MinLength` won't allow less than 8 characters so it doesn't really matter does it?